assets/charts/control-plane/kubernetes/templates/kube-controller-manager.yaml

```diff
@@ -60,6 +60,8 @@ spec:
         - --cluster-signing-cert-file=/etc/kubernetes/secrets/ca.crt
         - --cluster-signing-key-file=/etc/kubernetes/secrets/ca.key
         - --configure-cloud-routes=false
+        - --controllers=*,tokencleaner
+        - --experimental-cluster-signing-duration=1h
```

Comment:
I wouldn't enable token cleaning. If we clean them after an hour, it means that after 1h, worker pool will no longer be able to recover if instance gets recreated :/